rasa_nlu/train.py

```diff
@@ -62,6 +62,21 @@ def init():     # pragma: no cover
     return config
 
 
+def do_train_in_worker(config):
```

Comment:
The interest to keep the trainer and interpreter is that we can take advantage of the asynchronous model: once a successful training is done we can return the `persisted_path` value to the server process updating the current loaded model for a given agent/endpoint on the fly. Without this we lose part of what make the asynchronous server interesting.